sources/api/migration/migrations/v1.1.3/image-registry-mirrors/src/main.rs

```diff
@@ -0,0 +1,24 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddPrefixesMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added a new settings for configuring kubelet, `settings.image-registry.mirrors`
+/// We also added a new configuration template file for the Docker daemon
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
+        "settings.image-registry.mirrors",
```

Comment:
```suggestion
        "settings.image-registry",
```
To be clearer that we added this whole prefix.